src/Roslyn.Diagnostics.Analyzers/Core/AnalyzerReleases.Shipped.md

```diff
@@ -7,7 +7,6 @@ RS0001 | RoslyDiagnosticsPerformance | Warning | SpecializedEnumerableCreationAn
 RS0002 | RoslyDiagnosticsPerformance | Warning | SpecializedEnumerableCreationAnalyzer
 RS0005 | RoslyDiagnosticsPerformance | Warning | CodeActionCreateAnalyzer
 RS0006 | RoslyDiagnosticsReliability | Warning | DoNotMixAttributesFromDifferentVersionsOfMEFAnalyzer
-RS0013 | RoslyDiagnosticsPerformance | Disabled | DiagnosticDescriptorAccessAnalyzer
```

Comment:
Please don't remove rules from shipped file. Instead add a new "Remove Rules" table in Unshipped.md

Reply:
Ok sorry I wasn't sure about the how to.